vendor/github.com/koron/go-ssdp/README.md

```diff
@@ -46,7 +46,7 @@ for {
 }
 ```
 
-### Send bye when quiting
+### Send bye when quitting
```

Comment:
as @Poorunga mentioned, these are the vendor packages, should not be modified as this repo. see https://github.com/kubeedge/edgemesh/pull/461#issuecomment-1501912784

Reply:
addressed by https://github.com/kubeedge/edgemesh/pull/461/commits/a9cbdcb31915aa4625e7141271da67c2747dcc6c